Added more detail to contrib doc (#2905)
influxdata/docs-v2: content/influxdb/cloud/reference/flux/stdlib/contrib/_index.md

influxdb/v2.0/tags: [contributed, functions, package]
---

InfluxData supports user-contributed Flux packages and functions. We also recognize the importance of original contributors reviewing updates to their packages and functions.

> To support both contributors and our community at large, InfluxData requests reviews from original authors regarding all changes to the packages and functions.

Reviewer: ```suggestion
To support both contributors and our community at large, InfluxData will request a review by the original author on all proposed changes to packages and functions.
```

> User-contributed packages and functions are contributed and maintained by members of the InfluxDB and Flux communities. 
> 
> InfluxData supports user-contributed Flux packages and functions. We also recognize the importance of original contributors reviewing updates to their packages and functions. 
> 
> To support both contributors and our community at large, InfluxData requests reviews from original authors regarding all changes to the package.

Reviewer: ```suggestion
InfluxData supports user-contributed Flux packages and functions. We also recognize the importance of original contributors reviewing updates to their packages and functions. 

To support both contributors and our community at large, InfluxData requests reviews from original authors regarding all changes to the package.
```
